webfrontend/src/router/router.tsx

```diff
@@ -5,33 +5,39 @@ import { Register } from "../components/auth/register/register";
 import Home from "../pages/home/home";
 import Messages from "../pages/messages/messages";
 import Settings from "../pages/settings/settings";
+import Setup from "../pages/setup/setup";
+
 const router = createBrowserRouter([
-    {
+  {
+    path: "/",
+    element: <App />,
+    children: [
+      {
         path: "/",
-        element: <App />,
-        children: [
-            {
-                path: "/",
-                element: <Home />
-            },
-            {
-                path: "/login",
-                element: <Login />,
-            },
-            {
-                path: "/register",
-                element: <Register />,
-            },
-            {
-                path: "/messages",
-                element: <Messages />
-            },
-            {
-                path: "/settings",
-                element: <Settings />
-            }
-        ]
-    }
+        element: <Home />,
+      },
+      {
+        path: "/login",
+        element: <Login />,
+      },
+      {
+        path: "/register",
+        element: <Register />,
+      },
+      {
+        path: "/messages",
+        element: <Messages />,
+      },
+      {
+        path: "/settings",
+        element: <Settings />,
+      },
+      {
+        path: "/profile/setup",
+        element: <Setup />,
+      },
+    ],
+  },
```

Comment:
Keep the default indentation pls! Please use git to take file from another branch, not correcting the indentation yourself. Please see #14 comment for what I mean.